lib/requester.js

```diff
@@ -65,18 +65,30 @@ module.exports = function(requestObject, debug, cb){
       error.code = 400;
       return cb(error);
     }
-    var mapping = requestObject.mapping,
-    mapped;
-
-    if (typeof body === 'string' && response.headers.hasOwnProperty('content-type') && response.headers['content-type'].indexOf('application/json')>-1){
-      try{
-        body = JSON.parse(body);
-      }catch(err){
-        console.error('Error parsing JSON from response: ' + err);
+    var mapping = requestObject.mapping;
+    var mapped;
+
+    if (typeof body === 'string' && response.headers.hasOwnProperty('content-type')) {
+      var contentType = response.headers['content-type'];
+      if (contentType.indexOf('application/json') > -1) {
+        // JSON
+        try {
+          body = JSON.parse(body);
+        } catch (err) {
+          console.error('Error parsing JSON from response: ' + err);
+        }
+      } else if (contentType.indexOf('application/xml') > -1 || contentType.indexOf('text/xml') > -1) {
+        //XML response
+        try {
+          var xml2json = require("simple-xml2json");
```

Comment:
..had I know the solution would be this simple in the first place, I'm not sure why I didn't do it :D Nice work